integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -5436,10 +5438,76 @@ msgstr "Kontakt löschen"
 msgid "Deletion not possible"
 msgstr "Löschen nicht möglich"
 
+#: cms/templates/contacts/contact_from_email_and_phone.html
+msgid "Potential contact data"
+msgstr "Mögliche Kontaktdaten"
+
+#: cms/templates/contacts/contact_from_email_and_phone.html
+msgid ""
+"Here is the list of E-mail addresses and phone numbers that are embedded in "
+"contents. We recommend to replace them with a contact card so they can be "
+"centrally managed and updated in future."
+msgstr ""
+"Hier ist die Liste der E-Mail-Adressen und Telefonnummern, die in Inhalte "
+"eingebettet sind. Wir empfehlen, diese durch eine Kontaktkarte zu ersetzen, "
+"damit sie künftig zentral verwaltet und aktualisiert werden können."
+
```

Comment:
```suggestion
"Hier ist die Liste der E-Mail-Adressen und Telefonnummern, die in Inhalten "
"eingebettet sind. Wir empfehlen, diese durch eine Kontaktkarte zu ersetzen, "
"damit sie künftig zentral verwaltet und aktualisiert werden können."

```

Reply:
Ohhhh I just realized there's two ways to read this, you can ignore my comment. Sorry 😄